lib/moonshot/stack.rb

```diff
@@ -301,7 +309,7 @@ def make_tags
         default_tags << { key: @config.additional_tag, value: @name }
       end
 
-      default_tags + @config.extra_tags
+      (default_tags + @config.extra_tags + existing_tags).uniq!
```

Comment:
The issue I see here is that even when we do want to remove the tags this will always keep the existing tags.